sean/joke.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+  <head>
+      <meta charset="utf-8">
+      <title>Ew, David</title>
+      <!-- <link rel="preconnect" href="https://fonts.gstatic.com"> -->
+      <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;500&display=swap" rel="stylesheet">
+      <link href="joke_style.css" rel="stylesheet">
+  </head>
+
+  <body class="content">
+    <h1 class="header">omg look a joke</h1>
+    <main>
+      <p class= "joke">the mayor in my city just passed law that male best friends have to have lunch together at least once a week
+      </p>
+     <div class="img__wrap">
+        <img class="img__img" src="https://i.redd.it/aao5p6inltq51.jpg" width="365" height="477" alt= "david and patrick"/>
```

Comment:
Good work adding the `alt` attribute!! You could elaborate on what's happening _in_ the picture though so it really describes the content!

Reply:
Added deatiale description